iotdb-core/metrics/core/src/main/java/org/apache/iotdb/metrics/core/uitls/IoTDBMetricObjNameFactory.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.metrics.core.uitls;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+
+import java.util.Hashtable;
+
+public class IoTDBMetricObjNameFactory implements ObjectNameFactory {
+  private static final char[] QUOTABLE_CHARS = new char[] {',', '=', ':', '"'};
+  private static final Logger LOGGER = LoggerFactory.getLogger(IoTDBMetricObjNameFactory.class);
+
+  private IoTDBMetricObjNameFactory() {
+    // util class
+  }
+
+  @Override
+  public ObjectName createName(String type, String domain, String name) {
+    try {
+      ObjectName objectName;
+      Hashtable<String, String> properties = new Hashtable<>();
```

Comment:
Because the API provided by JMX requires us to use `HashTable` as parameter. We can see that in class `ObjectName`'s constructor.
<img width="718" alt="image" src="https://github.com/apache/iotdb/assets/78788603/83a19124-f048-4d46-bb9a-323d3fd2f0ad">

Reply:
got it